llama-index-core/llama_index/core/prompts/default_prompts.py

```diff
@@ -411,6 +411,7 @@
     "as the relevance score. The relevance score is a number from 1-10 based on "
     "how relevant you think the document is to the question.\n"
     "Do not include any documents that are not relevant to the question. \n"
+    "If the question is not answerable with the given documents, respond with the list of documents.\n"
```

Comment:
if we change to this prompt, what the LLM response will be like for such not relevant/answerable cases?